torchci/scripts/check_alerts.py

```diff
@@ -340,14 +342,14 @@ def map_job_data(jobNames: Any, shaGrid: Any) -> Dict[str, Any]:
     return jobData
 
 
-def is_job_not_pending_or_skipped(job: Any) -> bool:
+def is_job_failed(job: Any) -> bool:
     conclusion = job["conclusion"] if "conclusion" in job else None
-    return not (
-        conclusion is None
-        or conclusion == PENDING
-        or conclusion == NEUTRAL
-        or conclusion == SKIPPED
-    )
+    return conclusion is not None and conclusion != SUCCESS and conclusion != PENDING
+
+
+def is_job_skipped(job: Any) -> bool:
+    conclusion = job["conclusion"] if "conclusion" in job else None
+    return conclusion is None or conclusion == NEUTRAL or conclusion == SKIPPED
```

Comment:
Does the function name match the contents?

Doesn't `conclusion is None` mean the job is queued or recently started running (i.e. `pending`)?